### DISCUSSION TITLE
Disabling mass trasnfer in interReactiveTransferFoam

### DISCUSSION THREAD

**Discussion by Santhosh91000** [GeoChemFoam/GeoChemFoam#14] [Q&A] [answered]
Hello all,

Hope you're doing well!

I am using GeochemFoam and more specifically interReactiveTransferFoam solver. It is running fine on my computer but I am on the process of trying different things, and right now I am trying to disable the phase transfer by switching PhaseTransfer to ''off'' in ''thermoPhysicalProperties'' file in ''constant'' folder.

However, when I I do this, I am faced with the following problem:

''no phase transfer defined in constant/thermoPhysicalProperties:use interReactive  TransportFoam

--> FOAM FATAL ERROR:

FOAM aborting''

I didn't think I need to change further things in the solver or other files  since I have the possibility of turning ''off'' PhaseTransfer. But it seems I am wrong, please help me out figure out this issue.

Thanks for your time,
Sincerely,
Santhosh

**Comment by julienmaes** [comment, marked as answer]
Hi Santhosh, thanks for noticing this. Looks like a bug, I'll add it to the list. In the meantime, you can probably use the interReactiveTransportFoam solver, it does basically what you are trying to do.

Regards,

Julien

**Reply by Santhosh91000** [reply]
I see, thanks for confirming! 
I will work with interReactiveTransportFoam then.

Sincerely,
Santhosh
